README.md

```diff
@@ -34,35 +34,57 @@ You can either [clone the repository from GitHub](https://github.com/jtebert/kil
 
 ### Dependencies
 
-[HDF5](https://portal.hdfgroup.org/display/HDF5/HDF5), [SFML](https://www.sfml-dev.org/index.php), and [OpenMP](https://www.openmp.org/). Install on Ubuntu-like systems with: `sudo apt install libsfml-dev libhdf5-dev libomp-dev`
+[HDF5](https://portal.hdfgroup.org/display/HDF5/HDF5), [SFML](https://www.sfml-dev.org/index.php), and [OpenMP](https://www.openmp.org/). Install on Ubuntu-like systems with:
 
-[nlohmann/json](https://github.com/nlohmann/json): Add [`json.hpp`](https://github.com/nlohmann/json/releases/download/v3.5.0/json.hpp) to the `include/` folder. **Note:** This links to the most recent release (3.5) that has been verified for compatibility. You can find other [versions and release notes on Github](https://github.com/nlohmann/json/releases).
+    sudo apt install libsfml-dev libhdf5-dev libomp-dev
 
 ### Compile
 
-Create a static library at `bin/libKilosim.a`:
+To compile, copy and paste the following:
 
-    make
-or
+    mkdir build                                #Create a build directory
+    cd build                                   #Move into build directory
+    cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo .. #Configure for your system
+    make                                       #Build all
+    make install                               #Install files
+    cd ..                                      #Move back to source directory
 
-    make static
+Alternatively, you can run:
 
-Create an executable at `bin/kilosim` using the example `MyKilobot.cpp` and `test.cpp`:
+    make kilosim                        #Build library
+    make examples                       #Build examples
+    make kilosim_docs                   #Build documentation
```

Comment:
Should include `make clean`

Also, `make clean` doesn't clear the `bin/`

Reply:
Since we're doing an out-of-source build, you clean up by removing the `build` directory. `bin` and `lib` can go to. Edited the README to note this.